frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.test.tsx

```diff
@@ -12,6 +12,18 @@ import { render } from '@testing-library/react';
 import { noop } from 'lodash';
 import HeaderActions from './HeaderActions';
 import { mockParcel } from './InfoContent.test';
+import { useKeycloak } from '@react-keycloak/web';
```

Comment:
With the update to the keycloak package it required adding this to a number of tests.  Ideally I'd have time to clean it up and make it a centralized helper.